hack/setup-cluster.sh

```diff
@@ -562,10 +570,10 @@ pyroscope() {
 main() {
   if ! getopt -T > /dev/null; then
     # GNU enhanced getopt is available
-    parsed_opts=$(getopt -o e:k:n:r -l "engine:,k8s-version:,nodes:,registry" -- "$@") || usage
+    parsed_opts=$(getopt -o e:k:n:r -l "engine:,k8s-version:,nodes:,registry,fluentd-timeout:,csi-timeout:" -- "$@") || usage
   else
     # Original getopt is available
-    parsed_opts=$(getopt e:k:n:r "$@") || usage
+    parsed_opts=$(getopt e:k:n:r:f:c "$@") || usage
```

Comment:
By doing that, we mark `r` as requiring an argument, meanwhile `c`is not requiring it.
I think we should use `e:k:n:f:c:r ` instead of `e:k:n:r:f:c`, and the same applies to the previous getopt call.